src/comps/Cart.jsx

```diff
@@ -1,25 +1,22 @@
-import React, { useContext, useEffect, useState } from "react";
+import React, { useContext,useState } from "react";
 import { contextApi } from "../contextApi";
 import Prodact from "./Product";
+import cartCss from "../styles/cart.module.css";
+
 export default function Cart() {
-  const [pay, setPay] = useState(0);
+  const [refreshPage, setRefreshPage] = useState(false);
   const valContext = useContext(contextApi);
```

Comment:
Please don't use special state just to refresh the page, instead just change the value of the relevant state - it will cause the page to re render